website/docs/monitoring.md

```diff
@@ -0,0 +1,89 @@
+---
+id: monitoring
+title: "🧯 Monitoring"
+slug: /monitoring
+---
+
+![npm latest package](https://img.shields.io/npm/v/@monkvision/camera/latest.svg)
+
+## Modules overview
+
+The monitoring module provides different methods to monitor errors and measure metrics in the application.
+
+## Implementation Guild
+
+To implement `Monk` monitoring in an application, the entire application must be wrapped with a `MonitoringProvider`.
+
+``` javascript
+<MonitoringProvider config={config}>
+  <App />
+</MonitoringProvider>
+```
+
+The configuration options are listed below :
+
+## MonitoringConfig
+
+| **Config option**        | **Type** | **Type**                    | **Description**                              |
```

Comment:
Error in the second column title : it should be **Required** instead of **Type**